Remotion/Data/DomainObjects.Validation.IntegrationTests/Testdomain/StubStorageProvider.cs

```diff
@@ -116,5 +126,16 @@ private void CheckClassDefinition (ClassDefinition classDefinition, string argum
             StorageProviderDefinition.Name);
       }
     }
+
+    private void CheckDisposed ()
+    {
+      if (_disposed)
+        throw new ObjectDisposedException("StubStorageProvider", "A disposed StorageProvider cannot be accessed.");
+    }
+
+    private ArgumentException CreateArgumentException (string argumentName, string formatString, params object[] args)
+    {
+      return new ArgumentException(string.Format(formatString, args), argumentName);
+    }
```

Comment:
dead code?

Reply:
No, it is used om `CheckClassDefinition()`, which is used in `CreateNewObjectID`.